packages/build/config/nyc.common.json

```diff
@@ -0,0 +1,6 @@
+{
+  "include": ["packages/example-codehub/src/**", "packages/*/dist*/*"],
+  "exclude": ["packages/core/*/promisify.*"],
```

Comment:
This configuration is specific to loopback-next (include `packages/example-codehub`, exclude `promisify.*` from `core`), it does not belong to a shared/common config.